audio-effects/post_proc/ma_listener.c

```diff
@@ -44,7 +44,12 @@
 
 
 #define MA_SET_STATE "audio_hw_send_qdsp_parameter"
+
+#ifdef __LP64__
+#define HAL_VENDOR_PATH "/vendor/lib64/hw"
+#else
 #define HAL_VENDOR_PATH "/vendor/lib/hw"
+#endif
```

Comment:
For HW libraries, shouldn't they use `hardware_open()` or whatever it is?

So much NIH-ism in these HALs either way 😞

Reply:
it's just a mess. each file has its own style

Reply:
They really take the crown. Not just these files.